homeassistant/components/openweathermap/const.py

```diff
@@ -210,6 +212,7 @@
 FORECAST_SENSOR_TYPES = {
     ATTR_FORECAST_CONDITION: {SENSOR_NAME: "Condition"},
     ATTR_FORECAST_PRECIPITATION: {SENSOR_NAME: "Precipitation"},
+    ATTR_FORECAST_PRESSURE: {SENSOR_NAME: "Pressure"},
```

Comment:
Pressure is not part of the home assistant weather forecast.

https://developers.home-assistant.io/docs/core/entity/weather#forecast